src/components/Books/Books.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { SFC, useEffect, useState } from 'react';
+import IBook from '../../models/IBook';
+import { fetchBooks } from './api';
+import View from './BooksView';
+
+interface IBooksState {
+    books: IBook[];
+}
+
+const Books: SFC = () => {
+  const [books, loadBooks] = useState<IBook[]>([]);
```

Comment:
`useState` takes the default value for the state you are looking to create (`[]`, in this case), and returns a tuple with the current state (`books`) and a state update method (`loadBooks`).